spec/actions/updater_spec.rb

```diff
@@ -2,15 +2,15 @@
 
 require 'db/array_database'
 require 'db/file_database'
-require 'language_parser'
+require 'cli/language_parser'
```

Comment:
The language parser would be a generic purpose, top level class, so it doesn't need to be in the CLI folder